internal/provider/waypoint/resource_waypoint_add_on_definition.go

```diff
@@ -333,18 +337,6 @@ func (r *AddOnDefinitionResource) Create(ctx context.Context, req resource.Creat
 	}
 	plan.Labels = labels
 
-	if addOnDefinition.TerraformCloudWorkspaceDetails != nil {
```

Comment:
This was accidentally included in a merged PR so I am removing it so it's back to how it used to function.

Reply:
I think this one specifically is not quite right, the change in the PR that we should revert is [this](https://github.com/hashicorp/terraform-provider-hcp/pull/894/files#diff-514a4247ea434f616ba275ad89920b100f00d291f761cf4948bf3bab5da58283R336). By deleting the whole block, we can no longer read if the add-on definition's terraform cloud workspace details change, which isn't what we want.

Reply:
Gotcha, I'll make sure to update this back to how it was then!